### DISCUSSION TITLE
Does tailwind offer a Node.js API that I can run on strings of HTML?

### DISCUSSION THREAD

**Discussion by alexyuly** [tailwindlabs/tailwindcss#14711] [Help] [answered]
Does tailwind have an API that I can import in Node.js and then use to call a function on a string of HTML content and get back a string of compiled CSS content? It sounds like this functionality is part of the essence of tailwind. However, based on the documentation I only see two viable production-quality options: 1) static configuration using tailwind with your file system at build time, or 2) use the standalone CLI. Is there an option for using Tailwind at runtime (using Node.js, for a custom SSR application)?

**Comment by adamwathan** [comment, marked as answer]
This might be the sort of thing you're looking for:

https://github.com/tailwindlabs/tailwindcss/discussions/14636#discussioncomment-10895673

**Reply by alexyuly** [reply]
Yes! Thank you. This is what I'm looking for. The PostCSS config option for raw content is the part I was missing.
